requirements/test.txt

```diff
@@ -3,9 +3,10 @@ Brotli==1.0.9
 coverage==6.1.2
 cryptography==36.0.0; platform_machine!="i686" # no 32-bit wheels; no python 3.9 wheels yet
 freezegun==1.1.0
+# proxy.py==2.3.1
+https://github.com/abhinavsingh/proxy.py@master
```

Comment:
This branch is for stable releases. It is the same as v2.3.0: https://github.com/abhinavsingh/proxy.py/blob/master/requirements.txt. You may want to use `develop` instead. But then again, it's probably not something you need to change within this specific PR.
Plus I've been helping them revamp the whole packaging+CI/CD process entirely over the last month and there's a new version coming up soon. There's already an alpha release on PyPI created with the renewed packaging and fixed dependency pin if you _really_ want a bump:
```suggestion
proxy.py ~= 2.4.0a2
```
(`~=` is used to only allow pre-releases of this minor release; if you use `==` it'll implicitly use `--pre` in pip for all the versions of this package).